news/_posts/core-weekly/2019-07-31-coreweekly-week-29-2019.md

```diff
@@ -0,0 +1,129 @@
+* [#14605](https://github.com/PrestaShop/PrestaShop/pull/14605): Do not use encodeURIComponent with POST data, by [@PierreRambaud](https://github.com/PierreRambaud)
+* [#14613](https://github.com/PrestaShop/PrestaShop/pull/14613): Broken translation in module list page:, by [@matthieu-rolland](https://github.com/matthieu-rolland)
+* [#14656](https://github.com/PrestaShop/PrestaShop/pull/14656): Fix editing shop association in Category and Manufacturer. Thank you [@zuk3975](https://github.com/zuk3975)
+
+
+### Front office
+
+* [#12880](https://github.com/PrestaShop/PrestaShop/pull/12880): Add breadcrumb to the new products page. Thank you [@SebBareyre](https://github.com/SebBareyre)
+* [#12881](https://github.com/PrestaShop/PrestaShop/pull/12881): Add breadcrumb to the prices drop page. Thank you [@SebBareyre](https://github.com/SebBareyre)
+* [#12882](https://github.com/PrestaShop/PrestaShop/pull/12882): Add breadcrumb to the best sales page. Thank you [@SebBareyre](https://github.com/SebBareyre)
+* [#12883](https://github.com/PrestaShop/PrestaShop/pull/12883):  Add breadcrumb to the suppliers page. Thank you [@SebBareyre](https://github.com/SebBareyre)
+* [#13677](https://github.com/PrestaShop/PrestaShop/pull/13677): Submenu hides menu. Thank you [@YeLnatSs](https://github.com/YeLnatSs)
+* [#14214](https://github.com/PrestaShop/PrestaShop/pull/14214): Handle shoppingcart error returns properly - part 2, by [@matks](https://github.com/matks)
+* [#14558](https://github.com/PrestaShop/PrestaShop/pull/14558): Implement Schema ItemList for products listing & Fix Schema Product, by [@Progi1984](https://github.com/Progi1984)
+* [#14587](https://github.com/PrestaShop/PrestaShop/pull/14587): Fix issue when searching for long words in FO, by [@matthieu-rolland](https://github.com/matthieu-rolland)
+
+
+### Tests
+
+* [#14231](https://github.com/PrestaShop/PrestaShop/pull/14231): PHPUnit no expectation annotation. Thank you [@MathiasReker](https://github.com/MathiasReker)
+* [#14287](https://github.com/PrestaShop/PrestaShop/pull/14287): Behat tests for Category. Thank you [@sarjon](https://github.com/sarjon)
+* [#14593](https://github.com/PrestaShop/PrestaShop/pull/14593): First POM implementation, by [@SimonGrn](https://github.com/SimonGrn)
+* [#14612](https://github.com/PrestaShop/PrestaShop/pull/14612): Update docker for puppeteer, by [@boubkerbribri](https://github.com/boubkerbribri)
+* [#14725](https://github.com/PrestaShop/PrestaShop/pull/14725): Move travis scope from 5.6 to 7.1, by [@matks](https://github.com/matks)
+
+
+### Web services
+
+* [#8769](https://github.com/PrestaShop/PrestaShop/pull/8769): WS: Auto generate link_rewrite and do not require it when creating ca…. Thank you [@marekhanus](https://github.com/marekhanus)
+
+
+## Code changes in the "1.7.6.x" branch (for v1.7.6.0)
```

Comment:
`## Code changes in the "1.7.6.x" branch (for v1.7.6.0)`
=>
`## Code changes in the "1.7.6.x" branch (for v1.7.6.1)` 😉@ttoine

Reply:
ah ok. thanks. will update that, then